src/types/error/mysql-errors.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
To tak samo do usuniecia. I to akurat już bardziej nazwałbym jakimś typem :) w sensie, że tutaj to bym zostawił.